CHANGES.md

```diff
@@ -2,18 +2,29 @@
 
 ## __NEXT__
 
+### Features
+
+* filter: Add a new argument `--query-columns` allowing users to explicitly specify columns and data types for when automatic type inference behaves unexpectedly. [#1294][] (@victorlin)
```

Comment:
Seems also worth listing speed ups here (at least generally), if they hold up across the board in your testing. It's useful to know when upgrading might make something faster.

Reply:
👍 I'll add a changelog entry after doing more testing.

Reply:
Added to changelog:

> * filter, frequencies, refine: Performance improvements to reading the input metadata file. [#1294][] (@victorlin)
>     * For filter, this comes with increased writing times for `--output-metadata` and `--output-strains`. However, net I/O speed still decreased during testing of this change.
> 
> [#1294]: https://github.com/nextstrain/augur/pull/1294